packages/ui-components/src/components/windowedlist.ts

```diff
@@ -821,6 +854,12 @@ export class WindowedList<
     return this._isScrolling.promise;
   }
 
+  dispose(): void {
+    this._scrollbar.dispose();
+    this.layout.dispose();
```

Comment:
Unneeded as it is already taking care in `Widget`: https://github.com/jupyterlab/lumino/blob/a817cb2d05ac264403ad930fb9c038746c5393ff/packages/widgets/src/widget.ts#L74

```suggestion
```

Reply:
You're right. I'm rewriting this bit to try to move the scrollbar into layout after all. I think it's causing too many side-effects.